tests/sample/sample.vcxproj

```
@@ -238,6 +238,15 @@
       <TreatOutputAsContent Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">true</TreatOutputAsContent>
       <TreatOutputAsContent Condition="'$(Configuration)|$(Platform)'=='Release|x64'">true</TreatOutputAsContent>
     </CustomBuild>
+    <CustomBuild Include="tail_call_recursive.c">
```

```suggestion
    <CustomBuild Include="tail_call_multiple.c">
```
As noted in comment later on, it's just multiple tail calls, not really recursive.